test/sys/test_select.rs

```diff
@@ -31,6 +31,20 @@ fn test_fdset() {
     }
 }
 
+#[test]
```

Comment:
It is, otherwise rustc never calls this method (I just checked by putting a `panic!` inside). Maybe you're thinking of some other test file without harness?